pkg/security/api/types.go

```diff
@@ -0,0 +1,86 @@
+package api
+
+import (
+	"k8s.io/kubernetes/pkg/api/unversioned"
+	kapi "k8s.io/kubernetes/pkg/api"
+)
+
+// PodSecurityPolicySubjectReview checks whether a particular user/SA tuple can create the PodSpec.
+type PodSecurityPolicySubjectReview struct {
```

Comment:
> I think any user should be able to check whether a pod spec can be run in their namespace with any service account.

That is the object down here: `PodSecurityPolicyReview` which doesn't allow specifying generic users and groups.